db/entity/DiscordMember.ts

```diff
@@ -42,6 +42,9 @@ export class DiscordMember {
   @Column()
   customLink: string;
 
+  @Column({ nullable: true })
+  stellarWalletAddress: string;
```

Comment:
As for ethereum, no need to create a new field, just reuse starknetWalletAddress. Because we create one "DiscordMember" object for each wallet they connect, so it's already build to support several networks